semantic_steve/js/src/skill/get-placeable-coordinates/get-placeable-coordinates.ts

```
@@ -0,0 +1,45 @@
+    name: "getPlaceableCoordinates",
+    signature: "getPlaceableCoordinates()",
+    docstring: `
+          /**
+           * Gets the coordinates at which it is currently possible to place a block.
+           */
+        `,
+  };
+
+  constructor(bot: Bot, onResolution: SkillResolutionHandler) {
+    super(bot, onResolution);
+  }
```

Biome (1.9.4)

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)